products/libllbuild/BuildKey-C-API-Private.h

```diff
@@ -0,0 +1,33 @@
+//
+//  BuildKey-C-API-Private.h
+//  llbuild
+//
+//  Created by Benjamin Herzog on 01.08.19.
+//  Copyright © 2019 Apple Inc. All rights reserved.
+//
+
+#ifndef BuildKey_C_API_Private_h
+#define BuildKey_C_API_Private_h
+
+#include "llbuild/BuildSystem/BuildKey.h"
+
+namespace {
+using namespace llbuild::buildsystem;
+
+/// This class is used as a context pointer in the client
```

Comment:
I don't think this comment matches the code.

Reply:
It does. It gets handed out as an opaque `llb_build_key_t *` to the client which uses it as a context for the individual access functions in the C interface.